lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -50,6 +51,19 @@ export class RefreshTokenClient extends BaseClient {
         );
 
         responseHandler.validateTokenResponse(response.body);
+        if (response.body?.refresh_token) {
+            atsMeasurement?.endMeasurement({
+                success: true,
+                refreshTokenSize: response?.body?.refresh_token.length
```

Comment:
```suggestion
                refreshTokenSize: response?.body?.refresh_token.length || 0
```

Reply:
Then the else below isn't needed.

Reply:
Also, I think we would want this measurement not to end until `handServerTokenResponse` has returned below (i.e. do this logic in the `.then`/`.catch` of `responseHandler.handleServerTokenResponse`).

Reply:
Yes, makes sense. Have updated the file.